cfgov/v1/atomic_elements/organisms.py

```diff
@@ -535,6 +534,15 @@ class SimpleChart(blocks.StructBlock):
         'be referenced with dot notation: {"tooltip.shape": "circle"}',
     )
 
+    projected_months = blocks.IntegerBlock(
+        blank=True,
+        null=True,
+        min_value=0,
+        max_value=12,
+        help_text='How many months of the recent data are projected?',
+        required=False
```

Comment:
Couple formatting corrections. Use [`tox -e lint`](https://cfpb.github.io/consumerfinance.gov/python-unit-tests/#formatting) to lint our python files.

```suggestion
        help_text="How many months of the recent data are projected?",
        required=False,
```